src/aspire/source/xform.py

```diff
@@ -203,21 +203,21 @@ class FlipXform(Xform):
     """
     A `Xform` that applies phase flip to a stack of 2D images (as an Image object).
     """
-    def __init__(self, filters):
+    def __init__(self, filters_typ, filters_idx):
```

Comment:
I think we talked about this once - there's an `IndexedXform` that does what you're doing here, and it doesn't care what the actual `Xform` is. The problem with this approach here is that you'l end up duplicating this logic in every Xform where you want to apply different operations grouped by indices. Is this only place where you're storing the indices where you want to apply the Xform? We may open up a new issue into this.

Reply:
Current there is no call to `IndexedXform`. The main point of `FlipXform` is applying the a `LambdaFilter` to the original unique filter for flipping the filter values in k space. But I think it can be a derived class from `IndexedXform` if we try to use `IndexedXform`.

Reply:
ok..closing here. Opened a new issue.